src/tls/key_exchange.h

```diff
@@ -42,11 +35,15 @@ namespace tls
         throw std::logic_error(error_string(rc));
       }
 
+      crypto::EntropyPtr entropy = crypto::create_entropy();
```

Comment:
Curious: why is the entropy created in two places now?

Reply:
Because it's an mbedTLS implementation detail. Once I add the OpenSSL version, there will be no entropy objects in that respective implementation and/or the base class.